_posts/2018-08-01-cockpit-174.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Cockpit 174
+author: Gundersanne
+date: 2018-08-01
+tags: cockpit linux
+slug: cockpit-174
+category: release
+summary: VM pages for kubernetes, realmd install on demand
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.  Here are the release notes from version 174.
+
+### Kubernetes: VMs overview page and VM detail page
+
+The overview page shows all running virtual machines running on a cluster, with
+an option to filter by project. VMs can also be created on the overview page by
+supplying a json spec.
+
+![Kubernets VMs](/images/kubernetes-vms.png)
```

Comment:
This part isn't new though, I think -- just that you can click through the new details page, right?